### PR TITLE
Mobile Release v1.39.1

### PR DESCRIPTION
**Pull request by ceyhun**
## Description
Release 1.39.1 of the react-native-editor and Gutenberg-Mobile.

For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2748

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Review by cameronvoell — APPROVED** [review]
One comment on adding a section to the CHANGELOG.md. https://github.com/WordPress/gutenberg/pull/26327#discussion_r510587335

Interesting how the diff includes changes from the 1.39.0 release. I feel this is probably an artifact of how we tag releases on  release branches, and that these diffs will disappear when we merge master into this release branch before merging. Does that sound right to you @ceyhun? Let me know if you have any ideas on how we can keep these hotfix PRs from showing commits/ merges from the previous release, otherwise this PR looks good to go.

File: packages/react-native-editor/CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,10 @@ For each user feature we should also add a importance categorization label  to i
```
Comment (cameronvoell) [inline packages/react-native-editor/CHANGELOG.md:13]: Maybe we can put a 1.39.1 section here and update on the change included in the 1.39.1 betafix.

**Comment by ceyhun** [comment]
> Interesting how the diff includes changes from the 1.39.0 release. I feel this is probably an artifact of how we tag releases on release branches, and that these diffs will disappear when we merge master into this release branch before merging. Does that sound right to you @ceyhun? Let me know if you have any ideas on how we can keep these hotfix PRs from showing commits/ merges from the previous release, otherwise this PR looks good to go.

That's right! We can maybe have something like a `rnmobile/trunk` and keep it up-to-date on each release, but I'm not sure if it'll worth the effort.

**Comment by mchowning** [comment]
> That's right! We can maybe have something like a rnmobile/trunk and keep it up-to-date on each release, but I'm not sure if it'll worth the effort.

We did try doing that before and as I recall we had a lot of weird merge issues as a result of the squashed merges (changes would be both in the squashed release merge to `master` and on our `rnmobile/trunk` branch, which would be interpreted as conflicts).

**Comment by ceyhun** [comment]
After merging master into this branch, the changes look good now.
Will be merging this despite `End-to-End Tests / Admin - 1 (pull_request) ` failing, it's the same in master branch.

**Pull request merged**